text/1004-transactions-api.rst

```diff
@@ -651,16 +945,129 @@ There are few problems of this approach:
    network failures would be an issue
 
 
-Keep `transaction` as but Add a Helper for Retrying
----------------------------------------------------
+Keep `transaction` as is but Add a Helper for Retrying
+------------------------------------------------------
 
-The problem with this approach is that it hard to teach using `retry`
+The problem with this approach is that it hard to teach using ``retry``
```

Comment:
"The problem of adding a separate *transaction retry API* while keeping the current `Connection.transaction()` family of methods is that it will be hard to educate the users that they should almost always use the former. Most of the time transaction serialization and network errors are uncommon during the development and only surface when in production."